src/main/java/codeu/controller/RoutingEngine.java

```diff
@@ -0,0 +1,23 @@
+package codeu.controller;
+
+import codeu.model.data.BotRequest;
+
+/** This is the Routing Engine that will receive a BotRequest and pass it to the 
+ * appropriate BotHandler class*/
+public class RoutingEngine {
+
+  
+  /**
+   * primary routing function that passes the request to the appropriate Handler
+   * needs to be updated for each bot
+   */
+  public String routeCommand(BotRequest request) {
```

Comment:
Since the `BotHandler` doesn't return a `String` anymore, I don't think you want this to, either. I think you should actually throw an error probably, which the parser could handle, or you should log a warning that the command was not found at least.